### ISSUE TITLE
Typescript typings

### ISSUE THREAD

**Issue by anilanar** [dotcore64/react-stay-scrolled#101]
... would be great. Typings that we use in our project, introduced by @megheaiulian:

```
declare module 'react-stay-scrolled' {
  export interface UseStayScrolled {
    scroll: (position: number) => void;
    stayScrolled: () => void;
    scrollBottom: () => void;
    isScrolled: () => boolean;
  }
  export interface UseStayScrolledOptions {
    initialScroll: number = null;
    inaccuracy: number = 0;
    runScroll?: (offset: number) => void;
  }
  export default function useStayScrolled(
    domRef: React.MutableRef<HTMLElement>,
    options: UseStayScrolledOptions = {},
  ): UseStayScrolled;
}
```

This is probably not complete.

**Comment by perrin4869** [comment]
@anilanar I have zero expertise with typescript... Send me a PR and I'll merge :)

**Comment by perrin4869** [comment]
Hm... going through the issues, now that I got some TS experience under me I could do this, if it's still relevant haha

**Comment by perrin4869** [comment]
Published as `7.3.0`, thanks! Please tell me if there are any issues!
